pinot-common/src/main/java/org/apache/pinot/common/config/TagNameUtils.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.pinot.common.config;
 
 import org.apache.pinot.common.exception.InvalidConfigException;
+import org.apache.pinot.common.utils.CommonConstants;
```

Comment:
Not used import? Please reformat all the changed files

Reply:
Deleted.